content/guides/userguides/terra-export.md

```diff
@@ -0,0 +1,123 @@
+---
+path: "/guides/userguides/consumer-vignettes/export-to-terra"
+date: "2019-09-17"
+title: "Exporting HCA Data to Terra"
+subTitle: ""
+---
+
+# Exporting search results from the HCA Data Explorer to Terra
+
+In this tutorial, you'll learn how to send search results from the HCA Data
+Explorer to Terra and how to run a basic workflow with that data.
+
+We'll assume that you have some familiarity with the tools aforementioned.
+In particular, you'll want to be acquainted with the content in these
+tutorials:
+
+- [Accessing HCA Data and Metadata](quick-start-guide)
+
+You'll want to use Google Chrome for this tutorial; Terra has poorer support
+for other browsers.
+
+
+## Step one: finding BAM files with the HCA Data Explorer
+
+You can use the [HCA Data Explorer][explorer] to find data to export to Terra.
+The Data Explorer lists projects with data available for download from the Data
+Store and lets you filter the data for a number of attributes.
+
+Using the Data Explorer, select some data you're interested in. Choose anything
+that looks interesting - we'll be running a really simple workflow that
+generates MD5 checksums of files, so the type of data doesn't matter too much.
+When you've found what you're looking for, click on the big blue *Export
+Selected Data* button at the top right of the page. You'll see something like
+this:
+
+![The *Export Selected Data* button](_images/terra-export_button.png)
+
+Click on the *Export to Terra* button. You'll then see a page like this where
+you can select what kind of data to export:
+
+![Page for choosing data to export](_images/terra-choose_files.png)
+
+Again, choose whatever looks interesting.
+
+When you click the *Request Export* button, the Data Explorer will process your
+request, and you'll be redirected to Terra.
+
+
+## Step two: importing data to Terra and finding a workflow in Dockstore
+
+Select a workspace to import your selected data into. Once you have, you'll see
+a page like this, showing the data you just exported:
+
+![Terra page showing exported data](_images/terra-exported_data.png)
+
+Next, let's find a workflow to run with the data we've just exported. For this
+tutorial, we're looking for *dockstore-wdl-workflow-md5sum*, which will
+generate an MD5 checksum for a file (or files) that we provide. We'll need to
+import it from Dockstore. To do that, click on the *Workflows* tab at the top
+of the page, then on the big square *Find a Workflow* button. It'll look
+something like this:
+
+![Terra page showing workflows that can be added to workspace](_images/terra-workflows.png)
```

Comment:
```suggestion
![Terra page showing workflows that can be added to workspace](_images/terra_workflows.png)
```

Reply:
Filename in `_images` changed to match convention